docs/Analog.md

```diff
@@ -1,32 +1,95 @@
+6. Ensure the analog signal chain has minimal noise.
+
+As mentioned above, the analog input to the AMDC should be bipolar differential with the voltage within ±10V. Then, the analog front-end converts this into a single-ended pseudo-differential signal which is needed by the analog-to-digital converter (ADC). More information on the ADC input types and their definitions can be found [here](https://www.analog.com/media/en/technical-documentation/product-selector-card/2PB_sarinputtypesfb.pdf).
 
 ## Block Diagram / External Connections
 
-Pictorial representation of analog system, with blocks and descriptions of what they do.
+<img src="images/amdc-analog.svg" />
 
-## Schematics
+### Analog connectors
+The analog signal chain of the AMDC can receive up to 8 bipolar differential analog inputs (16 signals in total – 8 positive and 8 negative). There are 4 analog connectors (modular jacks) each receiving up to 2 differential inputs. There are also 4 ESD protection chips to protect the sensitive electronics from the damage due to ESD.
 
-Discussion related to schematics for analog subsystem:
-- Related screenshots
-- Hierarchy
+A table of the pin mappings for the analog connectors is shown below:
 
-## PCB Layout
+| Pin number | `Analog connector 1` | `Analog connector 2` | `Analog connector 3` | `Analog connector 4` |
+|------------|--------|-----|-----|-----|
+| 1 | `Analog P1` | `Analog P3` | `Analog P5` | `Analog P7` |
+| 2 | `Analog N1` | `Analog N3` | `Analog N5` | `Analog N7` |
+| 3 | +15V | +15V | +15V | +15V |
+| 4 | `GND` | `GND` | `GND` | `GND` |
+| 5 | -15V | -15V | -15V | -15V |
+| 6 | `GND` | `GND` | `GND` | `GND` |
+| 7 | `Analog P2` | `Analog P4` | `Analog P6` | `Analog P8` |
+| 8 | `Analog N2` | `Analog N4` | `Analog N6` | `Analog N8` |
+
+The location of these 4 analog connectors in the AMDC (stacked as 2 x 2) is shown below:
+
+<img src="images/amdc-analog-input-highlighted.svg" />
+
+In this picture, the top left connector (when looking from the right side) corresponds to the `Analog connector 1` in the block diagram. The top right connector is the `Analog connector 2`, the bottom left is the `Analog connector 3`, and the bottom right is the `Analog connector 4`.
+
+More detailed information on the analog connectors can be found in the [datasheet](https://media.digikey.com/pdf/Data%20Sheets/Amphenol%20PDFs/RJSAE_Brochure.pdf).
+
+### Difference amplifiers
+Analog input signals should be conditioned before transmitting to the ADC. The first stage of this conditioning is to decrease the voltage level. These are implemented using 4 [INA2143UA](http://www.ti.com/lit/ds/symlink/ina143.pdf) difference amplifier chips. Each chip has two op amps, totaling in 8 op amps each of which receives one differential analog input. These ICs require a ±15V supply which they receive from the AMDC. Each op amp is configured as a non-inverting amplifier as shown below:
+
+<img src="images/amdc-analog_op_amp.svg" />

```

Comment:
@Anvar-Khamitov Please rotate the bottom resistor to be vertical (like I had drawn in my hand sketch). That will make the image look more like the standard form of a non-inverting op amp with an offset.

![image](https://user-images.githubusercontent.com/7421348/83793374-1060b300-a662-11ea-958a-c40b92fea56b.png)

Reply:
Looks good.